src/components/Article.js

```diff
@@ -0,0 +1,22 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+export const Article = ({
+  title,
+  date,
+  text,
+}) => (
+  <article>
+    <h1>{title}</h1>
+    <span>{date}</span>
+    <p>{text}</p>
+  </article>
+);
+
+Article.protoTypes = {
+  article: PropTypes.shape({
+    title: PropTypes.string.isRequired,
+    date: PropTypes.string.isRequired,
+    text: PropTypes.string.isRequired,
+  }).isRequired,
```

Comment:
1. It's not `protoTypes`, it's `propTypes`
2. In your Article there are three separate props, not one shape. Fix proptypes.